### DISCUSSION TITLE
IO线程和业务线程在一个线程里面，应该把业务放到另外线程池里处理，另外回调的地方也没有拆包逻辑

### DISCUSSION THREAD

**Discussion by yijuwang** [chenshuo/muduo#605] [General]
GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, send it to mailing list:

muduo-library@googlegroups.com

Or visit:
https://groups.google.com/forum/#!forum/muduo-library

When file an issue muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

If you can't compile muduo, make sure you install `cmake` and `boost` from the
official package repository, e.g. `apt` or `yum`, before opening a bug.
Don't open a bug if you installed boost from a third-party source or
downloaded it by yourself, and couldn't compile muduo, thank you.

Also specify the exact environment where the issue occurs:

**Comment by chenshuo** [comment]
What's the question here?
